include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -282,13 +285,22 @@ class Radiation : public ISimulationPlugin
     {
         if(!notifyPeriod.empty())
         {
+            if(numJobs <= 0)
+            {
+                std::cerr << "'numJobs' must be '>=1' value is adjusted from" << numJobs << " to '1'." << std::endl;
+                numJobs = 1;
+            }
             // allocate memory for all amplitudes for temporal data collection
             tmp_result = new Amplitude[elements_amplitude()];
 
             /*only rank 0 create a file*/
             isMaster = reduce.hasResult(mpi::reduceMethods::Reduce());
 
-            radiation = new GridBuffer<Amplitude, DIM1 > (DataSpace<DIM1 > (elements_amplitude())); //create one int on GPU and host
+            /* Buffer for GPU results.
+             * The second dimension is used to store intermediate results if command
+             * line option numJobs is used.
+             */
+            radiation = new GridBuffer<Amplitude, 2> (DataSpace<2>(elements_amplitude(), numJobs));
```

Comment:
Would it not be:
```suggestion
            radiation = new GridBuffer<Amplitude, numJobs> (DataSpace<numJobs>(elements_amplitude(), numJobs));
```
?

Reply:
Not possible because the`2` must be a compile time value and `numJobs` and this `2` is not the same.
`GridBuffer<Amplitude, 2>` is saying that we operate on a matrix of results, so the old one dimensional buffer (one row/array) is now a two dimensional buffer.

Reply:
Okay - if it tsays 2 dimensional even for `numJobs > 2`, I am fine with this. But this increases the memory footprint always and not only by default.

Reply:
It increase the footprint by `numJobs` times. So does not increase for `numJobs == 1`, as N x 1 matrix takes the same space as an array of size N.

Reply:
Thanks for the clarification.